Lib/test/test_email/test_headerregistry.py

```diff
@@ -882,6 +882,16 @@ def content_disp_as_value(self,
             ('attachment; filename="Schulbesuchsbestättigung.pdf"'),
             ('Content-Disposition: attachment;\n'
              ' filename*=utf-8\'\'Schulbesuchsbest%C3%A4ttigung.pdf\n'),
+            ),
+
+        'parameter_value_with_fws_between_tokens': (
+            'attachment; filename="File Name With Spaces.pdf"',
+            'attachment',
+            {'filename': 'File Name With Spaces.pdf'},
+            [],
+            'attachment; filename="File Name With Spaces.pdf"',
+            ('Content-Disposition: attachment; '
+             'filename="File Name With Spaces.pdf"\n'),
```

Comment:
Not quite.  We need a test with something like "File =?utf-8?q?Name?= With Spaces.pdf".  That should have spaces around Name...we want to make sure we aren't removing spaces around encoded words that *aren't* next to other encoded words.

Reply:
Done.

Reply:
I'm not seeing the changes?

Reply:
You might be looking at the outdated diff, from a previous commit. 

There should be 4 commits in the PR, and your requested changes are in the last 2 commits.